src/bot.py

```diff
@@ -195,12 +199,13 @@ def on_create_game(self):
                 Logger.info(f"Please Enter the region ip and hot ip on config to use")
 
         # Run /nopickup command to avoid picking up stuff on accident
-        if not self._ran_no_pickup:
-            self._ran_no_pickup = True
-            if self._ui_manager.enable_no_pickup():
-                Logger.info("Activated /nopickup")
-            else:
-                Logger.error("Failed to detect if /nopickup command was applied or not")
+        self._ran_no_pickup = True
```

Comment:
Seems very much like unrelated change. Don't include changes that are not necessary for your feature.